### ISSUE TITLE
How to use Actions > Edit at License Browser

### ISSUE THREAD

**Issue by JustinWonjaePark** [fossology/fossology#1128]
Hi, I got confused while using Actions > Edit at License Browser.
I hope there is anybody who could help me.

Version of my FOSSology is 3.3.0.

When I remove licenses using "Edit", it works differently for files and folders.

For files it works as just what I expected.
1-1) Clicking "Edit" at a file
![image](https://user-images.githubusercontent.com/33045921/41885449-0062ff48-7933-11e8-8cb9-5cba7e3ce70f.png)
1-2) Removing license "LGPL-2.1+"
![image](https://user-images.githubusercontent.com/33045921/41885470-1bd11bde-7933-11e8-9ed5-6457d24859db.png)
1-3) Identified ("LGPL-2.1+" has been removed)
![image](https://user-images.githubusercontent.com/33045921/41885524-57c9c1d6-7933-11e8-9f7b-f9cbe3dfd4e6.png)

For a folder, I expected same thing(removed license) but it didn't worked that way.
2-1) Clicking "Edit" at a folder
![image](https://user-images.githubusercontent.com/33045921/41885561-7ef27a5a-7933-11e8-8bc7-40683df93a02.png)
2-2) ① Adding license "LGPL-2.1+" at "argp" ② Removing license "LGPL-2.1+" at "assert"
![image](https://user-images.githubusercontent.com/33045921/41885618-b2b07748-7933-11e8-9617-85d3248c3da4.png)
2-3) both folders are all cleared
![image](https://user-images.githubusercontent.com/33045921/41885631-c227ca3c-7933-11e8-89b5-ecae02810cf6.png)
2-4) difference (at "argp") : files without scanned result are identified as LGPL-2.1+
![image](https://user-images.githubusercontent.com/33045921/41885653-dfd7bd76-7933-11e8-9e6a-14f93bc90f9d.png)
2-5) difference (at "assert") : files without scanned result are identified without any license.
![image](https://user-images.githubusercontent.com/33045921/41885665-f42f75b6-7933-11e8-9d5a-84ff69880b62.png)

So, I suppose removing a license at a folder works similar with the "decider".
Is it the way it was supposed to work? or am I missing something?

If it is working as designed, is there anyway that I could remove a license scanned result at a folder?

**Comment by ag4ums** [comment]
@W-Park you can use bulk option to remove a license from a folder. (but you need to provide a suitable text that should be common to all the file) 
For more info : https://github.com/fossology/fossology/wiki/MonkBulk)

**Comment by shaheemazmalmmd** [comment]
Hello @W-Park,

**In the case 1 for add selected license folder :**
Licenses will get added to each and every file irrespective of license findings from agents.

so all the files get decided.

**In the case 1 for remove selected license folder :**

Here it removes all the licenses, and if there are any conflicts in license decisions  then it gets set to WIP where you can apply your decisions otherwise your decisions will be identified for all the files. This a expected behavior.

**Comment by W-Park** [comment]
@shaheemazmalmmd Sorry that I missed your comment! And thanks for explanation.
@mcjaeger Yes, that was useful. :)

**Comment by silverhook** [comment]
@mcjaeger, I agree that it would be useful to describe this behaviour in the documentation.

**Comment by shaheemazmalmmd** [comment]
User Documentation created. https://github.com/fossology/fossology/wiki/License-View
